### PR TITLE
implement version tracking

### PR DESCRIPTION
**Pull request by kdaily**
FYI only @thomasyu888!

I added `versionNumber` to track the version of the file that was validated or invalidated.

### CODE REVIEW

**Comment by thomasyu888** [comment]
Is this the first step to truly separating out validation and processing?

So your processing code would pull from this table the entity Id and version and do processing on that specific version?

**Comment by kdaily** [comment]
That is the end intention, yes! Right now it will record the version. I tested it so that it will always keep the most recent version only (effectively working the same way as original, just with the extra version column). It can also keep a record of all versions.

File: genie/input_to_database.py [hunk 1]
```diff
@@ -216,39 +217,43 @@ def _get_status_and_error_list(syn, fileinfo, valid, message, filetype,
...
+        input_status_list.append(record)
+
+    # else:
+        # # Send email the first time the file is invalid
+        # incorrect_files = ", ".join(filenames)
+        # incorrect_ent = syn.get(fileinfo['synId'][0])
+        # find_file_users = \
+        #     list(set([incorrect_ent.modifiedBy, incorrect_ent.createdBy]))
+        # usernames = ", ".join([
+        #     syn.getUserProfile(user)['userName']
```
Comment (thomasyu888) [inline genie/input_to_database.py]: Not going to send email about invalid files?

**Comment by thomasyu888** [comment]
Looks good to me.  It would also be nice if we could track the validity of each version uploaded (if that specific version was validated)

**Pull request merged**